install/debian-installation-guide.md

```diff
@@ -0,0 +1,47 @@
+# Install Kata Containers on Debian
+
+These install instructions are based on the Ubuntu packages, but we need to install an updated version of `librdb1`.
+Tested only on Debian 9 - Stretch (x86_64).
+
+1. Dependencies
+
+  We need a newer version of librdbd1 to satisfy requirements: https://packages.debian.org/sid/librbd1
+
+  add `unstable` repo to `/etc/apt/sources.list.d/unstable.list` sources list:
+  
+  ```
+  # for unstable packages
+  deb http://mirror.aarnet.edu.au/debian/ unstable main contrib non-free
```

Comment:
It isn't immediately obvious (by design almost ;), but all the installation guides are both human-readable *and* machine readable. This allows us to "execute" them using `kata-manager` - see:

- https://github.com/kata-containers/documentation/blob/master/install/installing-with-kata-manager.md

What this means is that all code blocks in the install docs *that the user needs to run*:

- must be specified in ` ```bash ` code blocks.
- must be non-interactive (to avoid hanging the CI waiting for user input).

If you need to show a code block as an example, just use a plain ` ``` ` code block.

Hence, here need to use a bash code block and  `sed` (or similar) to modify `/etc/apt/sources.list`.

More info:

- https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md

Reply:
That's a good policy, I will have another look over and correct these.

Reply:
Yes - we do it to guarantee that the installation guides always result in a working system ;)

Reply:
changes made